pep-9999.rst

```diff
@@ -0,0 +1,883 @@
+  license field.
+
+
+License in Language and Application packages
+--------------------------------------------
+
+- In Java, Maven POM [#maven]_ defines a licenses XML tag with a list of license
+  items each with name, url, comments and "disribution" type. This is not
+  mandatory and the content of each field is not specified.
+
+- JavaScript npm package.json [#npm]_ use a single license field with SPDX
+  license expression or the `UNLICENSED` id if no license is specified.
+  A license file can be referenced as an alternative using "SEE LICENSE IN
+  <filename>" in the single `license` field.
+
+- Rubygems gemspec [#gem]_ specifies either a singular license string for a list
+  of licenses strings. The relationship between multiple licenses in a list is
+  not specified. They recommend using SPDX license ids.
+
+- CPAN Perl modules [#perl]_ use a single license field wich is either a single
+  string or a list of strings. The relationship between the licenses in a list
+  is not specified. There is a list of support own license identifiers plus
+  these generic ids: open_source, restricted, unrestricted, unknown.
+
+- Rust Cargo [#cargo]_ specifies the use an SPDX license expession (v2.1) in the
+  license field. They also support an alternative expression synatx using slash-
+  separated SPDX license ids. There is a license_file field too. The crates.io
+  package registry [#cratesio]_ requires that either `license` or `license_file`
+  fields are set when you upload a package.
+
+- PHP Composer composer.json [#composer]_ uses a license field with an SPDX
+  License id or "proprietary". The License field is either a single string that
+  can use something which resemble SPDX license expression syntax with "and" and
+  "or" keywords; or this is a list of strings if there is a choice of licenses
+  (aka. a "disjunctive" choice of license).
+
+- NuGet packages [#nuget]_ were using only a simple license URL and are now
+  specifying to use an SPDX License expressions and/or the path to a license
+  file within the package. The NuGet.org repository states that they only
+  accepts license expressions that are `approved by the Open Source Initiative
+  or the Free Software Foundation.`
+
+- Golang has no provision for any metadata beyond dependencies. Licensing
+  information is left to community package managers.
+
+- Dart/Flutter spec [#flutter]_ recommends to use a single LICENSE file that
+  should contain multiple license texts each separated by a line with 80
+  hyphens.
+
+- JavaScript Bower [#bower]_ license field is either a single string or a list
+  of strings using either SPDX license identifier or a path or a URL to a
+  license file.
+
+- Cocoapods podspec [#cocoapod]_ license is either a single string or a mapping
+  with type, file an text keys. This is mandatory unless there is a LICENSE or
+  LICENCE fie provided.
+
+- Haskell Cabal [#cabal]_ specifies a single string with a list of accepted
```

Comment:
This is incorrect. `license` field is documented at https://cabal.readthedocs.io/en/latest/developing-packages.html#pkg-field-license

- it's not a list, it's proper *SPDX License Expression* (with additional `NONE`)

---

Cabal used to have own short list of licenses, but we moved to SPDX because
- more licenses
- expressions to combine them

Reply:
Thank you ++ for your review!
I fixed this in 3290f561b452d2d9a8830bf06187e187711115ff
I also added you name to the Acknowledgement section